lib/composer_messages_finder.rb

```diff
@@ -198,7 +198,7 @@ def check_get_a_room(min_users_posted: 5)
         .find_all { |uid| uid != @user.id && uid == reply_to_user_id }
 
     return unless last_x_replies.size == SiteSetting.get_a_room_threshold
```

Comment:
In this case I would prefer

```suggestion
    return if last_x_replies.size != SiteSetting.get_a_room_threshold
```

Not a hill I'm willing to die on though.

Reply:
I don’t have a strong opinion on that one, not sure whether it’s really better, though. But I can make the change anyway :grin: